CMake/telesculptor-external-fletch.cmake

```diff
@@ -23,18 +51,18 @@ ExternalProject_Add(fletch
     -DBUILD_SHARED_LIBS:BOOL=ON
     -Dfletch_BUILD_CXX11:BOOL=ON
     -Dfletch_BUILD_WITH_PYTHON:BOOL=${TELESCULPTOR_ENABLE_PYTHON}
-    -Dfletch_ENABLE_Boost:BOOL=ON
+    -Dfletch_ENABLE_Boost:BOOL=${TELESCULPTOR_ENABLE_FLETCH_BOOST}
     -DBoost_SELECT_VERSION:STRING=1.65.1
     -Dfletch_ENABLE_Caffe:BOOL=OFF
     -Dfletch_ENABLE_Caffe_Segnet:BOOL=OFF
-    -Dfletch_ENABLE_Ceres:BOOL=ON
+    -Dfletch_ENABLE_Ceres:BOOL=${TELESCULPTOR_ENABLE_FLETCH_CERES}
     -Dfletch_ENABLE_CppDB:BOOL=OFF
     -Dfletch_ENABLE_Darknet:BOOL=OFF
     -Dfletch_ENABLE_Darknet_OpenCV:BOOL=OFF
-    -Dfletch_ENABLE_Eigen:BOOL=ON
-    -Dfletch_ENABLE_FFmpeg:BOOL=ON
+    -Dfletch_ENABLE_Eigen:BOOL=${TELESCULPTOR_ENABLE_FLETCH_EIGEN}
+    -Dfletch_ENABLE_FFmpeg:BOOL=${TELESCULPTOR_ENABLE_FLETCH_FFMPEG}
     -DFFmpeg_SELECT_VERSION:STRING=3.3.3
-    -Dfletch_ENABLE_GDAL:BOOL=ON
+    -Dfletch_ENABLE_GDAL:BOOL=${TELESCULPTOR_ENABLE_FLETCH_GDAL}
     -Dfletch_ENABLE_GEOS:BOOL=ON
     -Dfletch_ENABLE_GFlags:BOOL=OFF
     -Dfletch_ENABLE_GLog:BOOL=ON
```

Comment:
I also don't think we use GLOG directly.  I know we build it for Ceres, but possibly for something else in Fletch too.  Maybe we should tie building GLOG to building Ceres?